lib/Common/Memory/HeapAllocator.h

```diff
@@ -130,6 +130,14 @@ struct HeapAllocator
     static HeapAllocator Instance;
     static HeapAllocator * GetNoMemProtectInstance();
 
+private:
+    HANDLE m_privateHeap;
```

Comment:
Thanks for catching that! I previously had this as a static member and initialized it as a static field but when I changed it I forgot to move the initialization to the constructor. I'll update shortly.